ebuild-writing/variables/text.xml

```diff
@@ -153,6 +153,33 @@ for these variables when writing ebuilds.
 		satisfied by <c>BDEPEND</c>, typically executable build tools.
     </ti>
   </tr>
+  <tr>
+    <ti><c>MERGE_TYPE</c></ti>
+    <ti>
+      The type of package that is being merged. Possible values are:
+      <c>source</c> if building and installing a package from source,
+      <c>binary</c> if installing a binary package,
+      <c>buildonly</c> if building a binary package without installing it.
+    </ti>
+  </tr>
+  <tr>
+    <ti><c>REPLACING_VERSIONS</c></ti>
+    <ti>
+      A whitespace-separated list of all versions (<c>PVR</c>) of this package
+      that are being replaced (uninstalled or overwritten) as a result of this
+      install. It is a list, not a single optional value, to handle pathological
+      cases such as installing <c>foo-2:2</c> to replace <c>foo-2:1</c> and
+      <c>foo-3:2</c>. Available in <c>pkg_preinst</c> and <c>pkg_postinst</c>.
+    </ti>
+  </tr>
+  <tr>
+    <ti><c>REPLACED_BY_VERSION</c></ti>
+    <ti>
+      The single version (<c>PVR</c>) of this package that is replacing us,
+      if we are being uninstalled as part of an install, or an empty string
```

Comment:
```suggestion
      from the perspective of the version being uninstalled as part of an install, or an empty string
```

Reply:
Hm, but then the "us" in the line before should also be replaced. How about:
"The single version (<c>PVR</c>) of this package that is replacing this ebuild's version as the result of an update, or an empty string otherwise."

Reply:
instead of saying `ebuild's version`, would it be better to say `the version provided by this ebuild` or `the version associated with this ebuild` ? Also, is there a particular motivation behind choosing `update` in the earlier part of the sentence and `uninstall` at the end.

Reply:
Right, "update" doesn't catch it, because it could also being a replacement by the same version.

"The single version (`PVR`) of this package that is replacing the version provided by this ebuild, if it is being uninstalled as part of an install. An empty string otherwise, i.e., if it is being uninstalled without replacement."

Reply:
It is a bit like in this classic from Morris Kline, "Mathematics in Western Culture": :smile:
> When a twelfth-century youth fell in love he did not take three paces backward, gaze into her eyes, and tell her she was too beautiful to live. He said he would step outside and see about it. And if, when he got out, he met a man and  broke his head—the other man's head, I mean—then that proved that his—the  first fellow's—girl was a pretty girl. But if the other fellow broke *his* head—not his own, you know, but the other fellow's—the other fellow to the second fellow, that is, because of course the other fellow would only be the other fellow to him, not the first fellow who—well, if he broke his head, then *his* girl—not the other fellow's, but the fellow who *was* the—
Look here, if A broke B's head, then A's girl was a pretty girl; but if B broke A's head, then A's girl wasn't a pretty girl, but B's girl was.